### PR TITLE
fix(DB/Item): Fix ProcChance of Charred Twillight Scale Trinket

### LINKED ISSUE #11335: [Item] Triggered Spells do not Proc
### Current Behaviour

Triggered Spell Proc 75465 from Item 54572 Charred Twilight Scale do not Proc

https://wotlkdb.com/?item=54572

Triggered Spell Proc 71602 from Item 50353 Dislodged Foreign Object do not Proc 

https://wotlkdb.com/?item=50353

### Expected Blizzlike Behaviour

Both items have a 10% Proc Chance to Triggered Spell

### Source

_No response_

### Steps to reproduce the problem

add items 54572 and 50353
Test Proc Chance from this items

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/azerothcore/azerothcore-wotlk/commit/a35e411fdd86f3b44e7b5f1a36b1ba2d43887a68

### Operating system

Windows 10 x64, Ubuntu 20

### Custom changes or Modules

_No response_

### PR DESCRIPTION
**Pull request by Meramaf**
## Changes Proposed:

-  Spell 71602 - Item - 50353 - Proc - 10 - Icecrown 25 Normal Caster Trinket 1 Base
-  Spell 71645 - Item - 50348 - Proc - 10 - Icecrown 25 Heroic Caster Trinket 1 Base
-   Spell 71845 - Item - 49992 - Proc - 2 - Icecrown 25 Normal Caster Weapon Proc
-   Spell 71846 - Item - 50648 - Proc - 2 - Icecrown 25 Heroic Caster Weapon Proc
-   Spell 72419 - Item - 50399/50400 - Proc - 10 - Icecrown Reputation Ring Healer Trigger
-   Spell 75465 - Item - 54572 - Proc - 10 - Chamber of Aspects 25 Nuker Trinket
-   Spell 75474 - Item - 54588 - Proc - 10 - Chamber of Aspects 25 Heroic Nuker Trinket  

## Issues Addressed:

- Closes #11335 

## Tests Performed:

- Tested
- 

## How to Test the Changes:

1. Add Item 54572 and 50353
2. Test the ProcChance
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by pklloveyou** [comment]
image
They all have the same mistake

**Comment by Meramaf** [comment]
> image They all have the same mistake

Yes that's right, thanks corrected everything ;)

**Comment by pklloveyou** [comment]
It works, but their trigger frequency is too high

**Comment by pklloveyou** [comment]
image
I tried to delete the above code without changing the database. Everything is normal

**Comment by Nefertumm** [comment]
Is there any source for these changes? Where did you get those values?

**Comment by Maelthyrr** [comment]
> Is there any source for these changes? Where did you get those values?

Probably here, one comment speak about 10%: https://www.wowhead.com/item=54572/charred-twilight-scale#comments

**Comment by Maelthyrr** [comment]
Work, but do we stick to a single point of view in a wowhead comment? Maybe someone is able to bring more sources?

**Comment by Meramaf** [comment]
All Procc Prozents default from wowhead.

**Comment by Maelthyrr** [comment]
can you link your sources pls ?

**Comment by Meramaf** [comment]
i also looked in original dbcs and compared. 

https://wotlkdb.com/?spell=71602

https://wotlkdb.com/?spell=71645

https://wotlkdb.com/?spell=71845

https://wotlkdb.com/?spell=71846

https://wotlkdb.com/?spell=72419

https://wotlkdb.com/?spell=75465

https://wotlkdb.com/?spell=75474

**Comment by Maelthyrr** [comment]
> i also looked in original dbcs and compared.
> 
> https://wotlkdb.com/?spell=71602
> 
> https://wotlkdb.com/?spell=71645
> 
> https://wotlkdb.com/?spell=71845
> 
> https://wotlkdb.com/?spell=71846
> 
> https://wotlkdb.com/?spell=72419
> 
> https://wotlkdb.com/?spell=75465
> 
> https://wotlkdb.com/?spell=75474

Oh  great, thanks :)

File: data/sql/updates/pending_db_world/rev_1650656505854517188.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1650656505854517188');
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1650656505854517188.sql:1]:
```suggestion
```

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by Zazezizozuzy** [comment]
I believe this over-corrected the issue and caused many of these items to proc too often. Spell Proc 71602 from Item 50353 Dislodged Foreign Object for example did not proc previously. Leading to the file change for this ticket: 
DELETE FROM `spell_proc_event` WHERE `entry` = '71602';

Removing that removed the proc cooldown (45000/45 seconds). The Heroic version still remains in spell_proc_event under entry 71645. That item retains the correct 45 second cooldown. I correctly this manually be reinserting a record for 71602. Let me know if I should open a new ticket for this.

**Pull request merged**